### ISSUE TITLE
SETTING_AUTH_LDAP_GROUP_TYPE and import statement not passed through properly the entrypoint script

### ISSUE THREAD

**Issue by tobiasjj** [zulip/docker-zulip#252]
To make LDAP group authentication for an Active Directory work, I have to add the following to the `settings.py` file:
```
from django_auth_ldap.config import ActiveDirectoryGroupType
AUTH_LDAP_GROUP_TYPE = ActiveDirectoryGroupType(name_attr="cn")
```

However, trying to add 
```
SETTING_AUTH_LDAP_GROUP_TYPE: ActiveDirectoryGroupType(name_attr="cn") 
```
 to the `docker-compose.yml` file, results in an error upon LDAP requests. The value `ActiveDirectoryGroupType(name_attr="cn")`  is added to `settings.py` as a string and the proper import statement for the class `ActiveDirectoryGroupType` is missing.

Setting the following in `docker-compose.yml` solves the issue:
`ZULIP_CUSTOM_SETTINGS: 'from django_auth_ldap.config import ActiveDirectoryGroupType\nAUTH_LDAP_GROUP_TYPE = ActiveDirectoryGroupType(name_attr="cn")'`

I suggest adding rules to process `AUTH_LDAP_GROUP_TYPE` in `entrypoint.sh` and automatically import missing classes.

**Comment by timabbott** [comment]
Yeah, we should probably just import the full set of things that are useful in `prod_settings_template.py` to avoid this.  @hackerkid FYI.
